src/slice.rs

```diff
@@ -128,38 +128,52 @@ impl From<Vec<u8>> for SlicePayload {
     }
 }
 
+/// Helper function for [`create_random_slice_payload`].
+/// Creates a Vec of [`Transaction`], binencodes it, and returns the resulting byte vector.
+/// The number of transactions used and their lens are picked so that the resulting byte vector is of size `desired_size`.
+//
+// NOTE: This function is terribly hacky and making assumptions about how bincode works.
+fn create_txs(desired_size: usize) -> Vec<u8> {
+    let mut rng = rng();
```

Comment:
I usually explicitly refer to `rand::rng()` to avoid confusion with the variables called `rng`.